src/picasso-definition/components/out-of-bounds/index.js

```diff
@@ -70,6 +83,9 @@ export default function createOutOfBounds({ models, context }) {
           },
         }),
         beforeRender: ({ size }) => {
+          windowSizeMultiplier = Math.min(size.height, size.width) / 300;
```

Comment:
What is this 300 for? Maybe put it to a const with a name explaining it's purpose...

Reply:
It's from old scatter plot when calculating point size. I tried to ask Quan why 300, he didn't reply. I think it is sort of windows size base number to get window size multiplier and then use to calculate point size. I add it as a constant with name `WINDOW_SIZE_BASE`.